src/hooks/useTransactionSteps.ts

```diff
@@ -90,15 +91,24 @@ const useTransactionSteps = ({ transaction, onComplete }: Props) => {
       return isCompleted ? TransactionStatusEnum.COMPLETED : TransactionStatusEnum.IN_PROGRESS;
     };
 
+    // 1. We wait until the block transaction gets finalized / transaction.block < source.bestBlockFinalized
```

Comment:
I don't think it has to be strictly greater. if `transaction.block == source.bestBlockFinalized` it is already finalized.

Reply:
The comment still needs an update.

Reply:
updated.